workspaces/jenkins/plugins/jenkins-backend/package.json

```diff
@@ -23,8 +21,23 @@
     "directory": "workspaces/jenkins/plugins/jenkins-backend"
   },
   "license": "Apache-2.0",
+  "exports": {
```

Comment:
We don't need this change and the one below as we aren't really using the Alpha export.